Cargo.toml

```diff
@@ -79,36 +77,3 @@ panic = "unwind"
 
 [features]
 runtime-benchmarks=["cli/runtime-benchmarks"]
-service-rewr= [
```

Comment:
This doesn't look like this should be removed should it?

Reply:
This is promoting the service rewrite to be the service, which means that as there are no longer two services, we no longer need the feature to choose the rewrite.